rst_files/discrete_dp.rst

```diff
@@ -597,9 +642,10 @@ What happens if the agent is more patient?
 
 .. code-block:: julia
 
-    g_2 = SimpleOG(β=0.99)
+    g_2 = SimpleOG(β=0.99);
+    matrices_2 = QR_matrices(g_2);
```

Comment:
I don't like this name.  Call it what it is, not the data structure.  `transition_matrices(...)` for example.
Also, I would prefer either splatting directly into the Q and R matrices, or forgetting the named tuple and just returning multiple arguments.

Notice
```
julia> using Parameters

julia> nt = (a = 2, b = 2)
(a = 2, b = 2)

julia> @unpack a, b = nt
(a = 2, b = 2)

julia> a
2

julia> b
2

julia> f() = (a = 2, b = 2)
f (generic function with 1 method)

julia> a, b = f()
(a = 2, b = 2)

julia> a
2

julia> b
2
```
i.e. you can return a named tuple into multiple arguments without an unpack.  But if you are only using a simple function once or twice, you can just return the tuple.